server/controllers/configurationsController/index.js

```diff
@@ -0,0 +1,69 @@
+import ConfigModel from '../../models/ConfigModel'
+
+const ConfigurationsController = {
+  create: async (req, res) => {
+    try {
+      const { appDb } = req.app.locals
+      await ConfigModel.save(appDb, req.body)
+
+      return res
+        .status(200)
+        .json({ data: { message: 'Configuration Created' } })
```

Comment:
I think we should be returning the saved configuration here.

Reply:
[2a243ef](https://github.com/AMP-SCZ/dpdash/pull/305/commits/2a243ef36a4f2e61ac2f5a1fccb5c52868e18464)